test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -2391,7 +2391,7 @@ def convert_to_cli(self, attrs, op=None, hostname=None, dflt_conf=True,
                                       self.platform == 'craysim' or
                                       self.platform == 'shasta'):
                 v = v.translate({ord(c): "\\" +
-                                 c for c in r"~`!@#$%^&*()[]{};:,./<>?\|-=_+"})
+                                 c for c in r"~`!@#$%^&*()[]{};:,./<>?\|-=+"})
```

Comment:
Not sure but as per qsub code, we don't escape -N value while sending to server, so I am guessing this escaping is needed for python->shell when PTL runs qsub command, if that's true then we have `@`, `%`, `:`, `,`, `.`,`/`,`-`,`=`,`+` also in list which I don't think need escape in shell

Reply:
Hi @hirenvadalia, above code is for PTL where it runs on remote hosts. Where qsub command runs from a mom or a client.  In this case we need escape character to be appended for special characters.

Reply:
@hirenvadalia As discussed offline, I have verified characters you have mentioned by giving in qsub -N option. Allowed characters are "_ , . , + , - ". I have removed them from special character list.